.github/labeler.yml

```diff
@@ -12,5 +12,8 @@ CMake:
 conda:
   - 'conda/**'
 
+cpp:
+  - 'cpp/**'
```

Comment:
We don't currently have a `cpp` folder in the rmm repo. This should be `include/` unless we want to reorganize the code.

Reply:
or rather should include `include/`, `tests/` and `doxygen/`

Reply:
Agree.